apps/dashboard/app/views/batch_connect/sessions/connections/_resolution_field.html.erb

```diff
@@ -0,0 +1,75 @@
+<div id="<%= id.to_s + "_group"%>" class="form-group">

```

Comment:
This file appears to be using tabs instead of spaces for whitespace. Please fix.